packages/react-ai/package.json

```diff
@@ -42,7 +42,8 @@
     "typecheck": "tsc --noEmit"
   },
   "peerDependencies": {
-    "aws-amplify": "^6.3.2",
+    "@aws-amplify/api-graphql": "4.3.0",
```

Comment:
```suggestion
    "@aws-amplify/api-graphql": "^4.3.0",
```

Reply:
Same question as https://github.com/aws-amplify/amplify-ui/pull/5796#discussion_r1763639952, in the main package.json we specify an exact version of `aws-amplify` which in turn uses exact versions of `@aws-amplify/api-graphql` and `@aws-amplify/api-core`, but our peer dep of `aws-amplify` uses a range of versions. So it seems like we would end up with multiple versions whenever `aws-amplify` updates. 

Should we update our devDep in the main package.json to use a range instead?

Reply:
Discussed offline and updated `aws-amplify` dep in top-level package.json to use a relative version